test/ejabberd_tests/tests/muc_SUITE.erl

```diff
@@ -1493,9 +1521,155 @@ admin_member_list(Config) ->
         escalus:assert_many(Preds, escalus:wait_for_stanzas(Bob, 2)),
 
         %% Kate receives her and Bob's presence
-        escalus:assert_many(Preds, escalus:wait_for_stanzas(Kate, 2))
+        escalus:assert_many(Preds, escalus:wait_for_stanzas(Kate, 2)),
+
+        %% Now Kate requests member list, and doesn't get it because she is neither owner, admin or moderator
+        %% and the room is configured without the roomconfig_getmemberlist setting
+        escalus:send(Kate, stanza_affiliation_list_request(
+            ?config(room, Config), <<"member">>)),
+        Error = escalus:wait_for_stanza(Kate),
+        escalus:assert(is_error, [<<"auth">>, <<"forbidden">>], Error)
   end).
 
+check_memberlist(Login, yes, Config) ->
```

Comment:
The `Login` var name can be misleading. In fact the variable contains record describing client's connection.